src/widgetastic_patternfly4/clipboardcopy.py

```diff
@@ -39,3 +35,22 @@ def fill(self, value):
 
     def copy(self):
         self.button.click()
+
+
+class ClipboardCopy(BaseClipboardCopy, Widget):
+    ROOT = ParametrizedLocator("{@locator}")
+    DEFAULT_LOCATOR = ".//div[contains(@class,'pf-c-clipboard-copy')]"
+
+
+class BaseClipboardCopyCompact(BaseClipboardCopy):
+    TEXT_LOC = ".//span[contains(@class, 'pf-c-clipboard-copy__text')]"
+    DEFAULT_LOCATOR = ".//div[contains(@class,'pf-c-clipboard-copy')]"
+    text = Text(locator=TEXT_LOC)
```

Comment:
This is already defined in `BaseClipboardCopy` and shouldn't be necessary - just changing TEXT_LOC.

Reply:
Well in the `BaseClipboardCopy` it's `TextInput` whereas here I use `Text`